fixtures/builder.py

```diff
@@ -40,6 +41,16 @@ def __init__(self, name):
 
         self.repository.status()
 
+    def __del__(self):
+        # Create a hash for the generated fixture.
+        hash_file = open(self.dir + "/hash.txt", "w")
+        file_contents = ""
+        for folder in ['client', 'server']:
+            file_contents = file_contents + folder + "-hash:" + dirhash(self.dir + "/" + folder, 'md5') + "\n"
```

Comment:
Why can't we just hash up the entire fixture directory, both `client` and `server`, in a single hash, before we create `hash.txt`?

Reply:
that makes sense. I think this was before I figured out we needed to delete all the hash files first.

I also moved the opening of the hash file till after the hash is create so we don't have the empty hash.txt file in the directory.